domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/EntityToResponseDTOMapper.java

```diff
@@ -171,4 +179,41 @@ private AuthorisationToAccessControlListMapper() {
                 .collect(Collectors.toList());
         }
     }
+
+    // Mapper to call fieldTypeEntity listItems only for fixedList items not for all fields.
+    // It improves performance.
+    class ListItemsMapper {
```

Comment:
can we call this FieldTypeEntityMapper? Because it's mapping a FieldTypeEntity, which represents all field types not only the ListItems